.github/workflows/macos.yml

```diff
@@ -59,14 +59,6 @@ jobs:
           node-version: '16.13.1'
           cache: 'npm'
           cache-dependency-path: '**/package-lock.json'
-      - name: Cache npm dependencies
```

Comment:
was hitting an issue with this, will explore in a different ticket how to reimplement this. Could have been because of a bad svelte version but i'll readdress this later.